Putting proposed changes onto public repo for Rishabh.

DennisLee-DennisLee opened the pull request:
For AB#166189: [AKS] New troubleshooting article for Connectivity to external resources.

AmandaAZ committed:
AB#166891: Client Secret Key Renewal Procedure

AmandaAZ committed:
add a template

AmandaAZ committed:
Merge remote-tracking branch 'upstream/main' into branch-forCI166891

AmandaAZ committed:
Update toc.yml

Commit:
Update

Commit:
Update again

Commit:
Update welcome-azure-kubernetes.yml

padmajayaraman committed:
Created a new file

padmajayaraman committed:
Merge branch 'main' into 167482_padmajayaraman_sqlconnector

padmajayaraman committed:
added TOC entry

padmajayaraman committed:
Merge branch 'main' into 167482_padmajayaraman_sqlconnector

padmajayaraman committed:
added more text

padmajayaraman committed:
Merge branch 'main' into 167482_padmajayaraman_sqlconnector

padmajayaraman committed:
edited text

padmajayaraman committed:
updated text

padmajayaraman committed:
formatted text

padmajayaraman committed:
added an empty line

padmajayaraman committed:
indented code snippet

padmajayaraman committed:
corrected 2 spelling errors in TOC

"funtion" as "function"
"mody" as "modify"

padmajayaraman committed:
updated text

padmajayaraman committed:
fixed formatting issues

padmajayaraman committed:
fixed formatting

padmajayaraman committed:
Update support/sql/admin/error-33111-restore-issues-sql-connector.md

Co-authored-by: Haiying <63399488+HaiyingYu@users.noreply.github.com>

padmajayaraman committed:
Update support/sql/admin/error-33111-restore-issues-sql-connector.md

Co-authored-by: Haiying <63399488+HaiyingYu@users.noreply.github.com>

padmajayaraman committed:
Update support/sql/admin/error-33111-restore-issues-sql-connector.md

Co-authored-by: Haiying <63399488+HaiyingYu@users.noreply.github.com>

padmajayaraman committed:
Update support/sql/admin/error-33111-restore-issues-sql-connector.md

Co-authored-by: Haiying <63399488+HaiyingYu@users.noreply.github.com>

padmajayaraman committed:
Update support/sql/admin/error-33111-restore-issues-sql-connector.md

Co-authored-by: Haiying <63399488+HaiyingYu@users.noreply.github.com>

padmajayaraman committed:
Update support/sql/admin/error-33111-restore-issues-sql-connector.md

Co-authored-by: Haiying <63399488+HaiyingYu@users.noreply.github.com>

padmajayaraman committed:
Update support/sql/admin/error-33111-restore-issues-sql-connector.md

Co-authored-by: Haiying <63399488+HaiyingYu@users.noreply.github.com>

padmajayaraman committed:
Update support/sql/admin/error-33111-restore-issues-sql-connector.md

Co-authored-by: Haiying <63399488+HaiyingYu@users.noreply.github.com>

padmajayaraman committed:
Update support/sql/admin/error-33111-restore-issues-sql-connector.md

Co-authored-by: Haiying <63399488+HaiyingYu@users.noreply.github.com>

padmajayaraman committed:
Merge branch 'main' into 167482_padmajayaraman_sqlconnector

padmajayaraman committed:
formatted and edited text

padmajayaraman committed:
edited text

padmajayaraman committed:
updated text

padmajayaraman committed:
edited text

HaiyingYu committed:
Fix typo

Commit:
Update

yushinok committed:
Removing not-supported statement

Given Intune Engineering LT confirmation, we are removing this non supported statement in disconnected forests.

padmajayaraman committed:
Merge branch 'main' into 167775_padmajayaraman_configurepermissions_accessremotedata

padmajayaraman committed:
edited text

padmajayaraman committed:
formatted text

padmajayaraman committed:
corrected an error

shwetasohu committed:
Implemented as per 167585

removed 2 articles and redirected to the KB 5017383

DennisLee-DennisLee committed:
Putting proposed changes onto public repo for Rishabh.

padmajayaraman committed:
Merge branch 'main' into 167482_padmajayaraman_sqlconnector

padmajayaraman committed:
updated text

padmajayaraman committed:
updated text

ramakoni1 committed:
Update error-33111-restore-issues-sql-connector.md

Made minor changes. - can be merged and made live.

MythilyMakam committed:
Merge pull request #3370 from MicrosoftDocs/167482_padmajayaraman_sqlconnector

AB#167482 SQL Connector

shwetasohu committed:
url corrected and toc section deleted

Deland-Han committed:
Merge branch 'main' into live

Commit:
Merging changes synced from https://github.com/MicrosoftDocs/SupportArticles-docs-pr (branch live)

NiklasEMS committed:
Update 0x80004027-remotely-access-com-plus-object.md

Same thing is applicable for other later Windows OS also.

ramakoni1 committed:
Update configure-permissions-access-remote-data.md

Made a couple of changes for clarity.

Dalbau committed:
Update verify-if-web-service-is-correct.md

kelleyguiney22 committed:
adding card to cross link to core docs (#3379)

* adding card to cross link to core docs

* removed spaces

* fixing link

* updating link list type

* moving the new card to the top

Deland-Han committed:
Merge branch 'main' into live

Commit:
Merging changes synced from https://github.com/MicrosoftDocs/SupportArticles-docs-pr (branch live)

Commit:
Update

Commit:
Update

Commit:
Merge pull request #3356 from simonxjx/simon-3

AB#165329: Update ms.service and add ms.subservices to AKS articles

Commit:
Merge remote-tracking branch 'upstream/main' into simon-3

Commit:
AB#167811 Updating article to prepare for Exchange Online disabling basic authentication

Commit:
Merge remote-tracking branch 'upstream/main' into simon-3

MythilyMakam committed:
Merge pull request #3378 from MicrosoftDocs/167775_padmajayaraman_configurepermissions_accessremotedata

AB#167775 Configure permissions access remote data

shwetasohu committed:
redirect url adjusted

shwetasohu committed:
redirect url adjusted

shwetasohu committed:
redirect_document_id changed

trying to fix redirect for its functionality

Deland-Han committed:
Merge branch 'main' into live

Commit:
Merging changes synced from https://github.com/MicrosoftDocs/SupportArticles-docs-pr (branch live)

genlin committed:
Merge pull request #3343 from MicrosoftDocs/repo_sync_working_branch

Confirm merge from repo_sync_working_branch to main to sync with https://github.com/MicrosoftDocs/SupportArticles-docs (branch main)

genlin committed:
Merge pull request #3387 from MicrosoftDocs/main

Sync to live

MythilyMakam committed:
Merge pull request #3381 from MicrosoftDocs/shwetasohu_update_per_AB#167585

Implemented as per 167585

brittmsantos committed:
ms.topic update.

ramakoni1 committed:
Update resolve-blocking-problems-caused-lock-escalation.md

ramakoni1 committed:
Update support/sql/performance/resolve-blocking-problems-caused-lock-escalation.md

Co-authored-by: JosephPilov-msft <23519517+PiJoCoder@users.noreply.github.com>

ramakoni1 committed:
Update resolve-blocking-problems-caused-lock-escalation.md

Deland-Han committed:
Merge branch 'main' into live

Commit:
Merge pull request #3386 from simonxjx/simon-3

AB#167811 Updating article to prepare for Exchange Online disabling basic authentication

Deland-Han committed:
Merge branch 'main' into live

DennisLee-DennisLee committed:
AB#166881: Azure Linux CN extension installation failure (#3321)

* Troubleshooting checklist.

* Ready for review.

* Added to TOC.

* Word change.

* Update troubleshoot-install-failure-hpc-pack-linux-node-agent.md

Edit review per CI 166881

* Addressed Jerry's concerns.

Co-authored-by: Jerry Sitser <v-jsitser@microsoft.com>

kelleyguiney22 committed:
note added (#3384)

* note added

* formatting two-paragraph note

* edits based on review of preview site

* Update enable-support-tls-environment.md

* Update enable-support-tls-environment.md

Edit review per CI per CI 167688

Co-authored-by: Jerry Sitser <v-jsitser@microsoft.com>

divargas-msft committed:
Update kernel-related-boot-issues.md

This entire document has been recreated. The previous document is not accurate (some of the kernel panics it was making a reference to, are not associated with no boot scenarios) and it was very outdated (recommending to delete the VM and recreated, instead of contemplating things like Repair VM, OS disk swap and so on and so forth. This document is needed by the Azure Linux Guided Troubleshooter.

divargas-msft committed:
Create delete

divargas-msft committed:
Add files via upload

divargas-msft committed:
Delete delete

divargas-msft committed:
Delete GettingtoGrub-01.gif

divargas-msft committed:
Add files via upload

divargas-msft committed:
Update kernel-related-boot-issues.md

divargas-msft committed:
Update kernel-related-boot-issues.md

divargas-msft committed:
Update kernel-related-boot-issues.md

divargas-msft committed:
Update kernel-related-boot-issues.md

Commit:
Update

divargas-msft committed:
Update kernel-related-boot-issues.md

divargas-msft committed:
Update kernel-related-boot-issues.md

Deland-Han committed:
Merge branch 'main' into live

divargas-msft committed:
Update kernel-related-boot-issues.md

divargas-msft committed:
Update kernel-related-boot-issues.md

Commit:
Update invalid-client-secret.md

hamenon-ms committed:
Updated the IP address records for MEO

hamenon-ms committed:
adjusted language

blakedrumm committed:
Add new issue for: `The user name or password is incorrect` :new:

blakedrumm committed:
Fix formatting error

thesqlsith committed:
Adding Managed Instance failover syntax

karimelmel committed:
Update decommission-enterprise-certification-authority-and-remove-objects.md

https://learn.microsoft.com/en-us/windows/win32/seccertenroll/cng-key-storage-providers#microsoft-software-key-storage-provider

rishasi reviewed (commented):
Updating section "Operating system disk heavily used"

rishasi reviewed (commented):
Updated section for Troubleshoot connections to pods and services within an AKS cluster - Azure

kelleyguiney22 committed:
committing all vm-deploy slugs (#3372)

* committing all vm-deploy slugs

* adding vm cannot start stop subservice

* comitting some more subservice tags

* adding more subservice tags (17 more)

* adding more subservice tags

kelleyguiney22 committed:
adding subservice tags for VM topics (#3385)

* adding subservice tags for VM topics

* adding last vmcannotconnect tags

* committing this to see if the build succeeds

* adding more

* last of this batch for this pr (I think)

* finishing additions to 2nd pr

kelleyguiney22 committed:
adding subservice tags for pr 3 (#3391)

* adding subservice tags for pr 3

* error for vm-troubleshooting-tools?

* last batch for now (I think)

blakedrumm committed:
Move article under correct section: **Signed certificate verification operations was not successful**

blakedrumm committed:
Add indents

Commit:
Merge pull request #3388 from brittmsantos/BMSPR1-10-4

Bulk Metadata Update - ms.topic update.

Commit:
Merge pull request #3377 from yushinok/patch-1

Removing not-supported statement

Commit:
Merge remote-tracking branch 'upstream/main' into pr/3302

Commit:
Update

Commit:
Merge branch 'branch-forCI166891' of https://github.com/AmandaAZ/SupportArticles-docs-pr into pr/3302

lucciz01 committed:
Merge branch 'main' into approvals-email-ip-update

Deland-Han committed:
Merge branch 'main' into live

lucciz01 committed:
minor changes

Commit:
Update 0x80004027-remotely-access-com-plus-object.md

Commit:
Merge pull request #739 from NiklasEMS/patch-1

Update 0x80004027-remotely-access-com-plus-object.md

Commit:
Merge pull request #742 from karimelmel/patch-1

Update decommission-enterprise-certification-authority-and-remove-obj…

lucciz01 committed:
minor changes

Commit:
Update troubleshoot-unix-linux-agent-discovery.md

Commit:
Update reporting-services-installation-fails.md

Commit:
Merge pull request #3401 from simonxjx/simon-1

Update reporting-services-installation-fails.md

ChadH81 committed:
Update invalid-client-secret.md

lucciz01 committed:
Merge branch 'main' into approvals-email-ip-update

lucciz01 committed:
Merge remote-tracking branch 'upstream/main' into pr/740

lucciz01 committed:
Merge pull request #3394 from MicrosoftDocs/approvals-email-ip-update

Updated the IP address records for Approvals

HaiyingYu committed:
Merge pull request #3389 from MicrosoftDocs/ramakoni1-patch-7

Update resolve-blocking-problems-caused-lock-escalation.md

AmandaAZ committed:
editorial changes

AmandaAZ committed:
Merge remote-tracking branch 'upstream/main' into branch-forCI166891

lucciz01 committed:
minor changes

AmandaAZ committed:
update images

sevend2 committed:
Update error-2767-query-secondary-replica-fails.md

sevend2 committed:
Merge pull request #3396 from thesqlsith/patch-1

Adding Managed Instance failover syntax

Commit:
Update troubleshoot-unix-linux-agent-discovery.md

lucciz01 committed:
minor changes

Commit:
Merge pull request #3395 from blakedrumm/patch-2

Add new issue for: `The user name or password is incorrect` :new:

genlin committed:
Merge pull request #3302 from AmandaAZ/branch-forCI166891

AB#166891: Client Secret Key Renewal Procedure

Deland-Han committed:
Merge branch 'main' into live

Commit:
Merging changes synced from https://github.com/MicrosoftDocs/SupportArticles-docs-pr (branch live)

lucciz01 committed:
minor changes

lucciz01 committed:
minor changes to the structure

lucciz01 committed:
Merge pull request #3404 from lucciz01/lucci109

minor changes

AmandaAZ committed:
editorial changes-second time

AmandaAZ committed:
Update kernel-related-boot-issues.md

Deland-Han committed:
Merge branch 'main' into live

Commit:
Merging changes synced from https://github.com/MicrosoftDocs/SupportArticles-docs-pr (branch live)

AmandaAZ committed:
Update kernel-related-boot-issues.md

AmandaAZ committed:
Update kernel-related-boot-issues.md

Deland-Han committed:
Merge pull request #3400 from MicrosoftDocs/repo_sync_working_branch

Confirm merge from repo_sync_working_branch to main to sync with https://github.com/MicrosoftDocs/SupportArticles-docs (branch main)

Deland-Han committed:
Merge branch 'main' into live

lucciz01 committed:
update the TOC and ms.subservice value

ChadH81 committed:
Update kernel-related-boot-issues.md

rishasi reviewed (changes requested):
Pushing changes.

rishasi reviewed (changes requested):
Pushing changes

ChadH81 committed:
Update kernel-related-boot-issues.md

lucciz01 committed:
Merge pull request #3405 from lucciz01/lucci1010

Update the TOC and ms.subservice value

ChadH81 committed:
Update kernel-related-boot-issues.md

AmandaAZ committed:
Update kernel-related-boot-issues.md

HaiyingYu committed:
Merge pull request #3375 from HaiyingYu/hy0930

Fix typo

genlin committed:
Merge pull request #3393 from divargas-msft/patch-3

AB#168030: Update kernel-related-boot-issues.md

Deland-Han committed:
Merge branch 'main' into live

blakedrumm committed:
Update script / verbiage :scroll:

Changed the script to allow easier use by customers, also added error handling.

Changed applicable version from 1807 and 2019, to all of Operations Manager versions.

sevend2 committed:
Adds redirections

sevend2 committed:
MINOR UPDATES

Commit:
Update cannot-deploy-operations-manager-reports.md

Commit:
Update cannot-deploy-operations-manager-reports.md

Commit:
Update cannot-deploy-operations-manager-reports.md

Commit:
Merge pull request #3410 from blakedrumm/patch-3

Update script / verbiage :scroll:

blakedrumm committed:
Brought some new cards to the table :books:

Commit:
Merge pull request #3412 from blakedrumm/patch-2

Brought some new cards to the table :books:

Deland-Han committed:
Merge branch 'main' into live

HaiyingYu committed:
Merge pull request #3411 from sevend2/SevenArchive

AB#168026: Adds redirections

HaiyingYu committed:
Merge pull request #3414 from MicrosoftDocs/main

publish

lucciz01 committed:
Merge pull request #740 from Dalbau/patch-2

AB#168029: Update verify-if-web-service-is-correct.md

DennisLee-DennisLee committed:
Merge branch 'main' of https://github.com/MicrosoftDocs/SupportArticles-docs into v-dele-test-166189

DennisLee-DennisLee committed:
Incorporated Rishabh's changes.
